steps/appcenter-codepush-release-react-native/0.0.3/step.yml

```diff
@@ -0,0 +1,70 @@
+title: AppCenter CodePush Release React Native
+summary: |
+  Release a React Native update to AppCenter CodePush
+description: |
+  Utilise appcenter-cli command appcenter codepush release-react to release an update
+website: https://github.com/adrianchinghc/bitrise-step-appcenter-codepush-release-react-native
+source_code_url: https://github.com/adrianchinghc/bitrise-step-appcenter-codepush-release-react-native
+support_url: https://github.com/adrianchinghc/bitrise-step-appcenter-codepush-release-react-native/issues
+published_at: 2021-02-19T10:58:15.91619+08:00
+source:
+  git: https://github.com/adrianchinghc/bitrise-step-appcenter-codepush-release-react-native.git
+  commit: d1dd92711fbf9526327b0bc3910c9d2e669e4862
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- react-native
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: node
+  apt_get:
+  - name: node
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    description: |
+      Path where the React Native project is located.
+    is_required: true
+    title: React Native Project Root
+  react_native_project_root: $REACT_NATIVE_PROJECT_ROOT
+- app_id: $APPCENTER_APP_ID
+  opts:
+    description: |
+      AppCenter Application Identifier is a combination of your Organization/Owner Name and an Application Name.
+      For example: `Upstack-Studio/bitrise-android`
+    is_required: true
+    summary: AppCenter application identifier (Organization/Owner Name + App Name).
+    title: AppCenter App ID
+- api_token: $APPCENTER_API_TOKEN
```

Comment:
isn't it a sensitive input variable? 🤔  if so please follow the section here: https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#secret-environment-variables-in-steps

Reply:
hey @adrianchinghc could you please mark this input as sensitive?